src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java

```diff
@@ -41,10 +42,13 @@ public StandardLineResource(StandardLineService standardLineService) {
         return ResponseEntity.notFound().build();
     }
 
-    @GetMapping(value = "/topic/{topicUUID}/standardLine", produces = APPLICATION_JSON_UTF8_VALUE)
+    @GetMapping(value = "/topic/{topicUUID}/standardLine", produces = APPLICATION_JSON_VALUE)
     public ResponseEntity<GetStandardLineResponse> getStandardLinesForPrimaryTopic(@PathVariable UUID topicUUID) {
         StandardLine standardLine = standardLineService.getStandardLineForTopic(topicUUID);
-        return ResponseEntity.ok(GetStandardLineResponse.from(standardLine));
+        if (standardLine != null) {
```

Comment:
I'm fine if this doesn't get done but would this be better to flip this. Have the expected behaviour be the default return? Return earlier if not found? Just be easier to read the flow in my opinion...

Reply:
Don't particularly mind either way... I've changed given I was there changing stuff.